tests/packagedcode/data/plugin/help.txt

```diff
@@ -174,6 +174,20 @@ Package type:  composer
   description:       PHP composer lockfile
   path_patterns:    '*composer.lock'
 --------------------------------------------
```

Comment:
Run also this script: https://github.com/nexB/scancode-toolkit/blob/2388-support-conan/docs/scripts/regen_package_docs.sh to update available package parsers script at `docs/source/reference/available_package_parsers.rst`

Reply:
Done!